README.md

```diff
@@ -67,35 +67,73 @@ Note that the specification files primarily define the structure (syntax) of a p
+but they should not include support for new features.
+Major versions are reserved for significant changes in the format of the files,
+the structure of packages or the interpretation of the Package Spec.
+
+Packages must specify the specification version they are using. This is done via
+the `format_version` property in the package's root `manifest.yml` file. The value
+of `format_version` must conform to the semantic versioning scheme.
+
+Specifications are defined by schema files and semantic rules, some attributes or
+files will only be available since, or till a version.
+
+Note that some versions may include a pre-release suffix, `e.g. 1.4.0-alpha1`. This
+indicates that these versions are still under development and may be changed multiple
+times. These versions in development can be used in prerelease versions of
+packages, but breaking changes can still occur.
+Once the pre-relase suffix is removed, however, the specification at that version becomes
+immutable. Further changes must follow the process outlined below in _Changing a Specification_.
 
 ## Changing a Specification
 
-* Consider a proposal to change the specification in some way. The version number of the changed specification must be determined as follows:
-  * If the proposed change makes the specification stricter than it is at `x.y.z`, the new version number will be `X.0.0`, where `X = x + 1`. That is, we bump up the major version. 
-     * If the change is in a schema file, consider the `spec` the latest
-       version, and add a JSON patch in the `versions` section to support older
-       schemas.
-     * If the change is in semantic rules, add a constraint in the rule, so they only apply on
-       the indicated version range.
-     * Add a changelog entry in the `spec/changelog.yml` file in the section of this major.
-  * If the proposed change makes the specification looser than it is at `x.y.z`, the new version number will be `x.Y.0`, where `Y = y + 1`. That is, we bump up the minor version and create a new changelog section in the `spec/changelog.yml` file. Note that adding new, but optional, constraints to a specification is a change that makes a specification looser.
-  * If the proposed change does not change the strictness of the specification at `x.y.z`, the new version number will be `x.y.Z`, where `Z = z + 1`. That is, we bump the patch version.
-     * Apply the proposed changes to the existing specification under the `spec` folder.
-     * Set the root-level `version` property in the specification's root `spec.yml` file to `x.y.Z`.
-     * Add a changelog entry in the `spec/changelog.yml` file in the section for
-       this version.
+Consider a proposal to change the specification in some way. The version number
+of the changed specification must be determined as follows:
+
+  * If the proposed change introduces changes in the format of the files that
+    require manual changes in packages, the new version number will be `X.0.0`,
```

Comment:
Maybe it can be rephrased to not used "changes" 

Maybe something like ? 

- If the proposed change introduces alters in the format of the files that require manual adjustments in packages, the new version
- If the proposed change introduces impacts the format of the files that require manual adjustments in packages, the new version

Reply:
Rephrased to "If the proposed change modifies the format of the files in a way that require manual changes in packages..."